### PR TITLE
Remove io.fabric8.kubernetes.model.annotation.PrinterColumn

### LINKED ISSUE #5343: Remove io.fabric8.kubernetes.model.annotation.PrinterColumn for 6.9
### Is your task related to a problem? Please describe

Reminder task to remove `io.fabric8.kubernetes.model.annotation.PrinterColumn` when we release 6.9.

### Describe the solution you'd like

Remove the class.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bachmanity1**
## Description
resolves #5343.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] I Added [CHANGELOG](https://github.com/fabric8io/kubernetes-client/blob/main/CHANGELOG.md) entry regarding this change
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/main/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by andreaTP** [comment]
Thanks @bachmanity1 for the help here!

For the maintainers do not merge this PR until we plan for 6.9.

**Review by oscerd — APPROVED** [review]

**Comment by manusa** [comment]
Hi @andreaTP,

Shall we consider merging this one now?

**Comment by andreaTP** [comment]
If we go for 6.9 yes 👍 , thanks for the heads up @manusa

**Review by andreaTP — APPROVED** [review]

**Pull request merged**
